src/main/java/com/pkb/unit/Unit.java

```diff
@@ -216,10 +239,19 @@ private void handleDependencyTransition(Transition transition) {
         }
     }
 
+    /**
+     * @return true when all the units are in STARTED state which this unit depends on. False otherwise.
```

Comment:
Another one that sounds strange...
Maybe:
```suggestion
     * @return true when all the units which this unit depends on are in STARTED state. False otherwise.
```